examples/repl.rs

```diff
@@ -152,6 +152,24 @@ mod example {
     fn run_command(context: &mut Context, command: &str) -> Result<(), Box<dyn std::error::Error>> {
         let mut parts = command.trim().split(' ').map(str::trim);
         match parts.next() {
+            Some("h") | Some("help") => {
+                println!("\x1b[1mList of Commands\x1b[0m");
+                println!("\x1b[1mexec\x1b[0m-- Start the debugged program");
+                println!("\x1b[1mattach\x1b[0m -- Attach to a process outside of headcrab");
+                println!("\x1b[1mdetach\x1b[0m -- Detach from a previously attached process outside of headcrab");
+                println!("\x1b[1mkill\x1b[0m -- Kill execution of prgram being debugged");
+                println!("\x1b[1mstepi|si\x1b[0m -- Step one instruction");
+                println!("\x1b[1mcontinue|cont\x1b[0m -- Continue the program being debugged");
+                println!("\x1b[1mregisters|regs\x1b[0m read -- List registers and their content for the current stack frame");
+                println!("\x1b[1mbacktrace|bt\x1b[0m -- Print backtrace of stack frames");
+                println!("\x1b[1mdisassemble|dis\x1b[0m -- Print the disassbled source");
+                println!(
+                    "\x1b[1mlocals\x1b[0m -- Print all local variables of current stack frame"
+                );
+                println!("\x1b[1m_patch_breakpoint_function|_pbf\x1b[0m -- Patch the `pause` instruction inside a function called `breakpoint` to be a breakpoint");
```

Comment:
Please remove this from the help. This command will be removed once breakpoints are implemented.